src/nxt_php_sapi.c

```diff
@@ -65,6 +65,15 @@ typedef struct {
 } nxt_php_run_ctx_t;
 
 
+HashTable nxt_php_user_config_cache;
+
+
+typedef struct {
+    nxt_time_t  expires;
+    HashTable   *user_config;
```

Comment:
`user_config` lifetime exactly the same as the whole `user_config_cache_entry`. Maybe it makes sense to store `HashTable` not the pointer?

Reply:
I was inspired by implementations in uWSGI and PHP SAPIs (fpm and litespeed) which all are using the same structure. Maybe it's a good idea to keep it similar to other implementations?

Reply:
I think it is always good to have a reasonable explanation why pointer used instead of the structure regardless of inspiration source.